Symfony/CS/Fixer/PSR2/ElseifFixer.php

```diff
@@ -30,25 +30,31 @@ class ElseifFixer extends AbstractFixer
     public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
+        for ($index = 7, $count = count($tokens); $index < $count - 1; ++$index) {
```

Comment:
I really don't like that index juggling, I never know `why the hell it's 7 this time???`.
For real code-files it won't change the performance, but it's decreases readability comparing to simple foreach loop.